src/OpenGl/OpenGl_OcclusionQuery.cxx

```diff
@@ -0,0 +1,110 @@
+// Created on: 2024-02-20
+// Created by: Hossam Ali
+// Copyright (c) 2024 OPEN CASCADE SAS
+//
+// This file is part of Open CASCADE Technology software library.
+//
+// This library is free software; you can redistribute it and/or modify it under
+// the terms of the GNU Lesser General Public License version 2.1 as published
+// by the Free Software Foundation, with special exception defined in the file
+// OCCT_LGPL_EXCEPTION.txt. Consult the file LICENSE_LGPL_21.txt included in
+// OCCT distribution for complete text of the license and disclaimer of any
+// warranty.
+//
+// Alternatively, this file may be used under the terms of Open CASCADE
+// commercial license or contractual agreement.
+
+#include <OpenGl_OcclusionQuery.hxx>
+
+#include <OpenGl_Context.hxx>
+#include <OpenGl_GlCore15.hxx>
+
+IMPLEMENT_STANDARD_RTTIEXT(OpenGl_OcclusionQuery, OpenGl_Resource)
+
```

Comment:
Please apply format.path from the GitHub artefact to follow the coding rules.
And additionally remove "// function..." from that header